utils/goroutines.go

```diff
@@ -98,6 +98,7 @@ func StartGoroutine(monitor *models.Monitor) {
 							}
 						}
 					}
+					heartbeat.Timestamp = time.Now().UTC()
```

Comment:
timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP AT TIME ZONE 'UTC' NOT NULL,
adding this in the sql would have solved this, migration would required too.